tools/topology/topology1/sof-cavs-nocodec.m4

```diff
@@ -218,12 +218,12 @@ PIPELINE_PCM_ADD(sof/pipe-host-volume-playback.m4,
 	PIPELINE_PLAYBACK_SCHED_COMP_5)
 
 # Deep buffer playback pipeline 11 on PCM 3 using max 2 channels of PIPE_BITS.
-# Set 1000us deadline on core SSP2_CORE_ID with priority 0.
+# Set 10000us deadline on core SSP2_CORE_ID with priority 0.
 # TODO: Modify pipeline deadline to account for deep buffering
 ifelse(PLATFORM, `bxt',
 `PIPELINE_PCM_ADD(sof/pipe-host-volume-playback.m4,
 	11, 3, 2, PIPE_BITS,
-	1000, 0, SSP2_CORE_ID,
+	10000, 0, SSP2_CORE_ID,
```

Comment:
Just a note that this has been never tested on Zephyr and current zephyr_ll.c does not support long-period tasks.

Reply:
@kv2019i This part of the pipeline should not even be managed by a LL scheduler....It seems we are missing XL parts of the puzzle to deliver the multi-stream capability.

Reply:
I've filed a bug for this https://github.com/thesofproject/sof/issues/5965

Reply:
agree, #5965 has to be fixed before modifying our topologies

Reply:
I think this warrants more discussion. I don't see #5965 a blocker. The todo in scheduler needs to be addressed for sure. But for the specific case of this PR, if we have frontend pipelines with larger period sizes, that should not affect the scheduling on FW side. Each of the larger period "front end" pipelines should still produce 1ms worth of data for each 1ms tick. The main difference is that the host side (and DDR) is woken up 10/20x less often. For simple PCM buffering, this seems sufficient.

We do have true needs to address slower rate of scheduling, when we have FW side code that cannot be broken into ll/1ms chunks. For this EDF is available, but with Zephyr we could also use lower-priority LL threads and rely on OS scheduling (but then we need to address #5965 on SOF side).

Reply:
> Each of the larger period "front end" pipelines should still produce 1ms worth of data for each 1ms tick.

@kv2019i this is true but when processing larger data isnt it natural to expect a lag especially right after start or after pause release?

Reply:
@ranj063  Yes but that can be handled by mixer logic. E.g. if stream is started and it takes longer to produce data, mixer can just ignore that source (as it has not produce any data). Let's say the stream produces a 20ms chunk of data, then mixer starts consuming it 1ms at a time and by the time it runs out, the next 20ms chunk will be ready. So there's lag at the start, but this can be handled as a prefill and mixer can ignore the source until it start producing data.